web/src/App.css

```diff
@@ -1,40 +1,95 @@
-.App {
+body {
   text-align: center;
```

Comment:
It'd be better if all elements on whole page didn't have `text-align: center` - you won't need `text-align: left` everywhere.